roles/ups/templates/operator.yml.j2

```diff
@@ -0,0 +1,42 @@
+apiVersion: apps/v1
```

Comment:
@wei-lee I believe we are using this template because of the image streams. I think that we'll remove this soon. @aliok @grdryn do we know when we're going to do that?

Reply:
This looks to be for the operator itself, rather than the service that it will manage. We don't use imagestreams for the operator. Is your question in regards to somewhere else in this PR rather than this file? If so, Ali is actively working on the removal of imagestreams from the UPS controller.